src/qt/sendcoinsdialog.cpp

```diff
@@ -342,12 +342,17 @@ void SendCoinsDialog::on_sendButton_clicked()
     questionString.append(QString("<span style='font-size:10pt;font-weight:normal;'><br />(=%2)</span>")
         .arg(alternativeUnits.join(" " + tr("or") + "<br />")));
 
-    if (ui->optInRBF->isChecked())
+    questionString.append("<hr /><span>");
+    if (!ui->disableRBF->isChecked())
     {
-        questionString.append("<hr /><span>");
-        questionString.append(tr("This transaction signals replaceability (optin-RBF)."));
-        questionString.append("</span>");
+      questionString.append(tr("You can increase the fee later (signals Replace-By-Fee)."));
```

Comment:
Nit, indentation
Also perhaps add reference to BIP 125 again

Reply:
Will do.